apps/store/src/components/Header/HeaderStyles.tsx

```diff
@@ -0,0 +1,97 @@
+import styled from '@emotion/styled'
+import * as NavigationMenuPrimitive from '@radix-ui/react-navigation-menu'
+import { ArrowForwardIcon, ChevronIcon, CrossIcon, mq, theme } from 'ui'
+import { MENU_BAR_HEIGHT_DESKTOP } from './Header'
+
+export const focusableStyles = {
+  cursor: 'pointer',
+  '&:focus-visible': {
+    outline: `2px solid ${theme.colors.gray900}`,
+  },
+}
+
+export const Navigation = styled(NavigationMenuPrimitive.Root)({
+  backgroundColor: theme.colors.gray200,
+  fontSize: theme.fontSizes[5],
+  [mq.md]: {
+    fontSize: theme.fontSizes[3],
+  },
+})
+
+export const NavigationMenuPrimitiveItem = styled(NavigationMenuPrimitive.Item)(({ theme }) => ({
+  [mq.md]: {
+    padding: `0 ${theme.space[4]}`,
+  },
+}))
+
+export const NavigationMenuPrimitiveContent = styled(NavigationMenuPrimitive.Content)(() => ({
+  [mq.md]: {
+    position: 'absolute',
+    top: 'calc(MENU_BAR_HEIGHT_DESKTOP + 0.5rem)',
```

Comment:
<i class='graphite__hidden'>[Re: lines 30 to 30]<br/></i>

Pretty sure this needs to be interpolated to work properly...

```suggestion
    top: `calc(${MENU_BAR_HEIGHT_DESKTOP} + 0.5rem)`,
```<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/952?utm_source=unchanged-line-comment">Graphite</a>.</span>

Reply:
That's right, thank you!